web/src/popover_menus.js

```diff
@@ -103,7 +119,8 @@ export function any_active() {
         compose_control_buttons_popover_instance ||
         compose_enter_sends_popover_displayed ||
         message_actions_popover_displayed ||
-        is_starred_messages_visible()
+        is_starred_messages_visible() ||
```

Comment:
We made some changes in how we manage popover visibility, and it might be helpful for you to take a look at `any_active` function.

Reply:
Changed the logic of `topic_menu` to use `any_active` function.